src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Extensions/StartupExtensions.cs

```diff
@@ -119,5 +125,41 @@ public static void UseHttpsRequestScheme(this IApplicationBuilder app)
                     return next(context);
                 });
         }
+
+        public static IServiceCollection ConfigureOptions(this IServiceCollection services, IConfiguration configuration)
```

Comment:
why have you added this part?

Reply:
I have moved here all options configuration, because it was messy in Startup.